From cf5ebff46e5bdc6044d8bfa65a5def82069cf3ce Mon Sep 17 00:00:00 2001
From: Jacob Gilbert <mrjacobagilbert@gmail.com>
Date: Sun, 21 Feb 2016 08:50:58 -0800
Subject: [PATCH 12/19] gr_modtool: gr_modtool rename command attempts to move
 GRC XML files based on block name, however it creates them with the module
 name prefixed, and attempts to move the wrong files. This fixes that behavior

---
 gr-utils/python/modtool/modtool_rename.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gr-utils/python/modtool/modtool_rename.py b/gr-utils/python/modtool/modtool_rename.py
index 8f1b250..68ac785 100644
--- a/gr-utils/python/modtool/modtool_rename.py
+++ b/gr-utils/python/modtool/modtool_rename.py
@@ -168,7 +168,7 @@ class ModToolRename(ModTool):
         grcfile = './grc/' + module + '_' + old + '.xml'
         self._run_file_replace(grcfile, old, new)
         self._run_cmakelists('./grc/', old, new)
-        self._run_file_rename('./grc/', old, new)
+        self._run_file_rename('./grc/', module + '_' + old, module + '_' + new)
 
     def _run_cmakelists(self, path, first, second):
         filename = path + 'CMakeLists.txt'
-- 
2.1.4

